### ISSUE TITLE
Accessibility: ORCA screen reader is not announcing the whole suggested information when we are typing in the copilot chat box

### ISSUE THREAD

**Issue by formulahendry** [microsoft/vscode#212320]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.89.0
- OS Version: Windows 11 Enterprise 23H2 (OS build 22631.3296)

Copied from https://github.com/microsoft/vscode-azureapicenter/issues/121

"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**

### GitHub Tags
#A11yTCS; #A11yMAS; #A11ySev2; #WCAG1.3.1; #GH_VSCodeextensionforAPICenter_Linux_Apr2024; #DesktopApp; #VS Code Extension for API Center; #ORCA; #Linux; #FTP; #Element:Dropdown;

### Actual Result:
ORCA screen reader is not announcing the whole suggested information when we are typing in the copilot chat box

### Expected Result:
NVDA/Jaws should announce the whole suggested information when we are typing in the copilot chat box. for ex. if we are typing @apicenter in the copilot chat then screen reader should also announce Build, consumer and consume great APIs.

### Environment Details:
Application Name: VS Code Extension for API Center
OS: Windows 11 Enterprise 23H2 (OS build 22631.3296)
Screen reader: ORCA

### Repro Steps:

1. Start ORCA screen reader.
2. Create an API center in azure portal.
3. Open Visual studio Code.
4. Tab till Chat option in the left side panel and press enter.
5. Type @apicenter in chat box.
6. Verify whether the ORCA screen reader is not announcing the whole suggested information when we are typing in the copilot chat box.

### User Impact:
Screen reader will not know about the whole information if the screen reader is not announcing it.

### Attachment:
![121_A11y_VS Code extension for API Center_Copilot Chat_ScreenReader](https://github.com/microsoft/vscode-azureapicenter/assets/90901519/757202db-0600-4b34-8842-d4e63a33438f)

**Comment by meganrogge** [comment]
pls don't open duplicate issues /duplicate #212322
